app/Http/Controllers/UserController.php

```diff
@@ -126,17 +144,111 @@ public function editPassword(Request $request)
+            $archive_extension = config('mimetypes')[$image->getImageMimetype()];
+
+            $is_duplicated_archive_image_path = true;
+            while($is_duplicated_archive_image_path){
+                $archive_image_path = Str::random(rand(20, 50)).'.'.$archive_extension;
+                $is_duplicated_archive_image_path = User::where('archive_image_path', $archive_image_path)->exists();
+            }
+                
+            if(!is_null($image->getImageProperties("exif:*"))){
+                $image->stripImage();
+            }
+
+            Storage::put('public/archive_images/'.$archive_image_path, $image);
+            Storage::delete('public/archive_images/'.$user->archive_image_path);
+
+            $is_duplicated_thumbnail_image_path = true;
+            while($is_duplicated_thumbnail_image_path){
+                $thumbnail_image_path = Str::random(rand(20, 50)).'.webp';
+                $is_duplicated_thumbnail_image_path = User::where('thumbnail_image_path', $thumbnail_image_path)->exists();
+            }
+
+            if($archive_extension !== '.webp'){
+                $image->setImageFormat('webp');
+            } 
+            $image->resizeImage(200, 200, Imagick::FILTER_LANCZOS, 1);
+                        
+            Storage::put('public/thumbnail_images/'.$thumbnail_image_path, $image);                    
+            Storage::delete('public/thumbnail_images/'.$user->thumbnail_image_path);
+
+            $image->clear();
+
+            $user->thmbnail_image_path = $thumbnail_image_path;
+            $user->archive_image_path = $archive_image_path;
+        }
+
+        $user->name = $request->name;
+        $user->kana_name = $request->kana_name;
+        $user->gender = $request->gender;
+        $user->birthday = $request->birthday;
+        $user->phone_number = str_replace('-', '', $request->phone_number);
+        $user->postal_code = str_replace('-', '', $request->postal_code);
+        $user->prefecture = $request->prefecture;
+        $user->address = $request->address;
+        $user->block = $request->block;
+        $user->building = $request->building;
+        $user->save();
+
+        return to_route('users.complete', $user->id)->with(['is_updated_user_info' => true]);
+    }
+
+    public function updateEmail(EmailResetRequest $request, User $user)
+    {
+        $authentication = Authentication::where('email', $request->email)
+            ->where('status',  AuthenticationStatus::MAIL_SENT)
+            ->where('type', AuthenticationType::EMAIL_RESET)
+            ->where('expired_at', '>', now())
+            ->where('user_id', $user->id)
+            ->first();
+
         if(is_null($authentication)){
             return to_route('login_credential.create')->withErrors(['reset_error' => '無効なアクセスです。']);
         }
+        
+        $user->email = $request->email;
+        $user->save();
+        $authentication->status = AuthenticationStatus::COMPLETED;
```

Comment:
```suggestion
        $user->save();
        
        $authentication->status = AuthenticationStatus::COMPLETED;
```
処理の内容が分かれているので、改行入れておいてください。

Reply:
改行を追加致しました。
ご確認よろしくお願い致します。
app/Http/Controllers/UserController.php